_includes/footer.html

```diff
@@ -0,0 +1,80 @@
+<footer class="footer">
+  {% include footer-navigation.html %}
+  <div class="usa-identifier">
+    <section class="usa-identifier__section usa-identifier__section--masthead"
+      aria-label="{{ site.data.settings.identifier.accessible_labels.identifier }}">
+      <div class="usa-identifier__container">
+        <div class="usa-identifier__logos">
+          <a href="https://www.gsa.gov" class="usa-identifier__logo">
+            <img class="usa-identifier__logo-img" src="{{ site.baseurl }}/assets/img/gsa-logo.svg"
+              alt="{{ site.data.settings.identifier.accessible_labels.gsa_link_logo }}" />
+          </a>
+        </div>
+        <div class="usa-identifier__identity" aria-label="{{ site.data.settings.identifier.accessible_labels.description }}">
+          <p class="usa-identifier__identity-domain">Login.gov</p>
+          <p class="usa-identifier__identity-disclaimer">
+            {{ site.data.settings.identifier.disclaimer | markdownify | replace: '<p>', '' | replace: '</p>', '' }}
```

Comment:
[question] Are the replacements necessary? i don't see any p tags in the yml

Reply:
Oh wow, these replacements _are_ necessary. `markdownify` adds `p` tags around anything it inserts, so we go an extra level deep, which breaks some CSS (and is bad HTML).

Reply:
ah, ok, thanks for investigating